awsim_sensor_kit_launch/config/concatenate_and_time_sync_node.param.yaml

```diff
@@ -0,0 +1,22 @@
+/**:
+  ros__parameters:
+    debug_mode: false
+    has_static_tf_only: false
+    rosbag_replay: false
+    rosbag_length: 20.0
+    maximum_queue_size: 5
+    timeout_sec: 0.2
+    is_motion_compensated: false
+    publish_synchronized_pointcloud: true
+    keep_input_frame_in_synchronized_pointcloud: true
+    publish_previous_but_late_pointcloud: false
+    synchronized_pointcloud_postfix: pointcloud
+    input_twist_topic_type: twist
+    input_topics: [
+                    "/sensing/lidar/right/pointcloud_before_sync", # 0.05
```

Comment:
What does `0.05` mean?

Reply:
The 0.05 is the timestamp I observed from the pointcloud. (0.05, 0.15, 0.25, 0.35 ...)

Reply:
Sorry, could you provide a more precise explanation?

Reply:
If you run `ros2 topic echo /sensing/lidar/right/pointcloud_before_sync --field header`, the timestamp of pointcloud will show up. 

I noticed that the timestamp is in the format xxxxxxxxx.05xxxx and increases by 0.1 seconds per frame.

Reply:
From your comment below, seems that I couldn't get the timestamp of left and right